lib/hex_web/package.ex

```diff
@@ -187,11 +187,11 @@ defmodule HexWeb.Package do
       name_search = "%" <> name_search <> "%"
     end
 
-    desc_search = String.replace(search, ~r"\s+", " & ")
+    desc_search = String.replace(search, ~r"\s+", " | ")
 
       from var in query,
     where: ilike(var.name, ^name_search) or
-           fragment("to_tsvector('english', (?->'description')::text) @@ to_tsquery('english', ?)",
+           fragment("to_tsvector('english', regexp_replace((?->'description')::text, '/', ' ')) @@ to_tsquery('english', ?)",
```

Comment:
I think it is needed, I'll give some examples that can help us to clarify the whole scenario:

If we change the description before (via `regexp_replace`) the result is `true`, regardless of the operator:

```sql
milmazz=# select to_tsvector(regexp_replace('Elixir... WSDL/SOAP Services', '/', ' ')) @@ to_tsquery('WSDL | SOAP') as result;     
 result 
----------
 t
(1 row)

milmazz=# select to_tsvector(regexp_replace('Elixir... WSDL/SOAP Services', '/', ' ')) @@ to_tsquery('WSDL & SOAP') as result;
 result 
----------
 t
(1 row)
```

If we don't change the description, the result is `false`.

```sql
milmazz=# select to_tsvector('Elixir... WSDL/SOAP Services') @@ to_tsquery('WSDL | SOAP') as result;
 result 
----------
 f
(1 row)

milmazz=# select to_tsvector('Elixir... WSDL/SOAP Services') @@ to_tsquery('WSDL & SOAP') as result;
 result 
----------
 f
(1 row)
```

Finally, look the lexemes created by `to_tsvector` in this case. So, I think that's why in the previous 2 cases the lexemes created by `tsquery` does not match (e.g. `wsdl | soap` does not match with `wsdl/soap`):

```sql
milmazz=# select to_tsvector('Elixir binding to Detergent erlang library used to call WSDL/SOAP Services');
                                          tsvector                                           
---------------------------------------------------------------------------------------------
 'bind':2 'call':9 'deterg':4 'elixir':1 'erlang':5 'librari':6 'servic':11 'use':7 'wsdl/soap':10
(1 row)
```

Any advice?